unearthed-arcana/2017/20170206.xml

```diff
@@ -326,6 +326,11 @@
           <stat name="HP" value="1" level="18" />
           <stat name="HP" value="1" level="19" />
           <stat name="HP" value="1" level="20" />
+          <stat name="ac:stones durability:base" value="13" />
+          <stat name="ac:stones durability:constitution" value="constitution:modifier" />
+          <stat name="ac:stones durability" value="ac:stones durability:base" />
+          <stat name="ac:stones durability" value="ac:stones durability:constitution" />
+          <stat name="ac:calculation" value="ac:stones durability" bonus="calculation" equipped="[armor:none]" alt="Stone’s Durability (Stone Sorcerer)" />
```

Comment:
You can skip the extra setters `ac:stones durability:base` and `ac:stones durability:constitution` in this one, it's not going to change.

```
<stat name="ac:stones durability" value="13" />
<stat name="ac:stones durability" value="constitution:modifier" />
```
This is all you need.

And since the AC is something you get when use use the feature instead of something you always have, you can remove the `ac:calculation` stat. The calculated AC will be available from the sheet description when you use it.

Reply:
Great suggestion! I made that change.